modulefiles/build_cheyenne_gnu

```diff
@@ -15,13 +15,13 @@ module load ncarcompilers/0.5.0
 module load python/3.7.9
 module unload netcdf
 
-module use /glade/p/ral/jntp/GMTB/tools/hpc-stack-v1.2.0/modulefiles/stack
+module use /glade/work/epicufsrt/GMTB/tools/gnu/10.1.0/hpc-stack-v1.2.0/modulefiles/stack
 module load hpc/1.2.0
 module load hpc-gnu/10.1.0
 module load hpc-mpt/2.22
 
 module load srw_common
-module load g2/3.4.3
+#module load g2/3.4.3
```

Comment:
Feel free to remove these extra g2 module loads rather than just commenting them out...that was a temporary fix for a bug in the old hpc-stack build (https://github.com/NOAA-EMC/hpc-stack/issues/376)

Reply:
Okay. Good to know. Thanks!